components/drawer/index.en-US.md

```diff
@@ -39,6 +39,8 @@ A Drawer is a panel that is typically overlaid on top of a page and slides in fr
 | onClose | Specify a callback that will be called when a user clicks mask, close button or Cancel button. | function(e) | - |
 | afterVisibleChange | Callback after the animation ends when switching drawers. | function(visible) | - |
 | keyboard | Whether support press esc to close | Boolean | true |
+| footer | The footer for Drawer. | ReactNode | - | 4.0.0 |
```

Comment:
Remove `4.0.0` since all props is validate in v4

Reply:
> Remove `4.0.0` since all props is validate in v4

@zombieJ done